_resources/port1.0/group/crossgcc-1.0.tcl

```diff
@@ -261,6 +262,18 @@ proc crossgcc.setup {target version} {
         # fatal error: error in backend: ran out of registers during register allocation
         compiler.blacklist  {clang >= 421 < 422}
 
+        if { ${version} >= 11.0 } {
+            # https://trac.macports.org/ticket/29067
+            # https://trac.macports.org/ticket/29104
+            # https://trac.macports.org/ticket/47996
+            # https://trac.macports.org/ticket/58493
+            compiler.blacklist-append {clang < 800} gcc-4.0 *gcc-4.2 {llvm-gcc-4.2 < 2336.1} {macports-clang-3.[4-7]}
+
+            # https://build.macports.org/builders/ports-10.13_x86_64-builder/builds/105513/steps/install-port/logs/stdio
+            # c++/v1/functional:1408:2: error: no member named 'fancy_abort' in namespace 'std::__1'; did you mean simply 'fancy_abort'?
+            compiler.blacklist-append {clang < 1000}
```

Comment:
Would you be willing to open a new trac ticket for this?

There is something strikingly similar among the old tickets (listing further references):
https://trac.macports.org/ticket/59908

Reply:
The compile blacklist information was taken directly from the gcc11 port, so the same trac ticket of open would be relevant for this.

Reply:
Added a note that this section was taken from gcc11 Port.

Reply:
@kencu was there a ticket associated with the listed error?

Reply:
Not that I know of; looks like a buildbot discovery, fixed by a compiler bump, with no further digging in on the details at the time.

Reply:
@kencu makes sense I just figured you’d know more about it since compilers seem to be your thing. But I’m guessing since it’s resolved by a compiler bump it didn’t qualify for a ticket.

@mojca I’d added a note explaining the section was taken from the gcc11 Port.